lib/uploadcare/entity/file_metadata.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Uploadcare
+  module Entity
+    # This serializer is responsible for file metadata handling
+    #
+    # @see https://uploadcare.com/api-refs/rest-api/v0.7.0/#tag/File-metadata
+    class FileMetadata < Entity
+      client_service FileMetadataClient
+
+      class << self
+        def index(uuid)
+          ::Uploadcare::Client::FileMetadataClient.new.index(uuid).success
+        end
+
+        def show(uuid, key)
+          ::Uploadcare::Client::FileMetadataClient.new.show(uuid, key).success
+        end
+
+        def update(uuid, key, value)
+          ::Uploadcare::Client::FileMetadataClient.new.update(uuid, key, value).success
+        end
+
+        def delete(uuid, key)
+          '200 OK' if ::Uploadcare::Client::FileMetadataClient.new.delete(uuid, key).success?
```

Comment:
думаю, что тут лучше вертать ответ сервера, не строку. Так как он может измениться и в будущем нужен конечному юзеру

```suggestion
          ::Uploadcare::Client::FileMetadataClient.new.delete(uuid, key).success
```

Reply:
там просто `nil` возвращается, поэтому я решил лучше уж какую-то понятную инфу возвращать.

Reply:
давай тогда уж сделаем так:

```ruby
::Uploadcare::Client::FileMetadataClient.new.delete(uuid, key).success || '200 OK'
```
чтобы юзер хотя бы мог получить тело ответа, если оно есть